Restructure API namespace for Cy13 (#5053)
cypress-io/cypress-documentation: docs/api/commands/wait.mdx

| `requestTimeout`  | [`requestTimeout`](/guides/references/configuration#Timeouts)                                                                 | Overrides the global `requestTimeout` for this request. Defaults to `timeout`.           |
| `responseTimeout` | [`responseTimeout`](/guides/references/configuration#Timeouts)                                                                | Overrides the global `responseTimeout` for this request. Defaults to `timeout`.          |

### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)

- `cy.wait()` ignores the subject it was given, except to pass it through if
  given a `time` argument.

### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)

- `cy.wait()` has no implicit assertions.

### Timeouts [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Timeouts)

- When given a `time` argument, `cy.wait()` cannot time out.

> - When given an alias algument, `cy.wait()` can time out waiting for the request

Reviewer: ```suggestion
- When given an alias argument, `cy.wait()` can time out waiting for the request
```

Author: Applied, and to the next line as well. :+1:

>   to go out.
> - When given an alias algument, `cy.wait()` can time out waiting for the
>   response to return.

Reviewer: ```suggestion
  to go out or the
  response to return.
```

Author: I broke this out into two sentences because you can specify different request and response timeouts - it seemed like two different timeout options = two different ways it can time out.
